extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -575,7 +596,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
   /** @override */
   layoutCallback() {
-    const superReturnValue = super.layoutCallback();
+    const frameLoadPromise = super.layoutCallback();
```

Comment:
We should consider refactoring this so that we pass the loadPromise to onCreativeRender such that layoutCallback override is no longer required.

Reply:
Done.

Reply:
On second thought, as mentioned offline, this'll be done in a separate PR.